src/pages/docs/argo-cd/steps/update-application-image-tags.md

```diff
@@ -37,45 +38,48 @@ This step will execute on a worker of your choosing - if required it can run wit
 1. Specify the Container Images which are to be updated  in your Argo Application.
 `Note`: These packages can then be used in an [external feed trigger](/docs/projects/project-triggers/external-feed-triggers), such that your cluster is automatically updated when new image versions become available.
 
-
 ### Outputs
+
 The output section allows you to configure how changes are to be merged into your repository.
 
-1. Deployment Preview is an aid to help determine which instances, and which applications are going to be updated when executing this step
-   * More information can be found [here](/docs/argo-cd/steps/deployment-preview)
+1. "Argo CD Applications View" is an aid to help determine which instances, and which applications are going to be updated when executing this step
+   - Read more about the [annotated application view step](/docs/argo-cd/steps/annotated-application-view)
```

Comment:
Is it `Argo CD Applications View` or `annotated application view step` ?

Reply:
lets go with [Argo CD Applications View](/docs/argo-cd/steps/argo-cd-applications-view)